### PR TITLE
changes to remove shortcode array

### PR DESCRIPTION
**Pull request by ccfredbaum**
Remove Shortcode Class 
create Logger class, 
adjust all classes that use add_shortcode methods to use a constant add in class which goes to App as the defined value. 
remove unecessary use Shortcodes

### CODE REVIEW

File: my-video-room/class-sitedefaults.php [hunk 1]
```diff
@@ -78,7 +77,7 @@ public function init() {
 		if ( ! Factory::get_instance( RoomAdmin::class )->check_default_settings_exist() ) {
 			Factory::get_instance( Setup::class )->initialise_default_video_settings();
 		}
-		$this->add_shortcode( 'invitemenu', array( Factory::get_instance( MeetingIdGenerator::class ), 'invite_menu_shortcode' ) );
+		add_shortcode( self::SHORTCODE_TAG . 'invitemenu', array( Factory::get_instance( MeetingIdGenerator::class ), 'invite_menu_shortcode' ) );
```
Comment (alecsammon) [inline my-video-room/class-sitedefaults.php:80]: can the "invitemenu" go into the constant

`const SHORTCODE_TAG = App::SHORTCODE_TAG . '_invitemenu';`

File: my-video-room/library/class-logger.php [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+ */
+
+declare( strict_types=1 );
+
+namespace MyVideoRoomPlugin\Library;
+
+/**
+ * Class Logger
+ */
+abstract class Logger {
```
Comment (alecsammon) [inline my-video-room/library/class-logger.php:15]: I don't think this should be abstract

File: my-video-room/module/security/shortcode/class-securityvideopreference.php [hunk 3]
```diff
@@ -29,7 +30,7 @@ class SecurityVideoPreference extends Shortcode {
 	 * Provide Runtime
 	 */
 	public function init() {
-		$this->add_shortcode( 'choose_security_settings', array( $this, 'choose_security_settings_shortcode' ) );
+		add_shortcode( self::SHORTCODE_TAG . 'choose_security_settings', array( $this, 'choose_security_settings_shortcode' ) );
```
Comment (alecsammon) [inline my-video-room/module/security/shortcode/class-securityvideopreference.php:33]: can `choose_security_settings` got into the constant

`const SHORTCODE_TAG = App::SHORTCODE_TAG . '_choose_security_settings';`

File: my-video-room/module/sitevideo/class-mvrsitevideo.php [hunk 4]
```diff
@@ -79,7 +77,7 @@ public function de_activate_module() {
 	 */
 	public function init() {
 		$site_video_controller = Factory::get_instance( MVRSiteVideoControllers::class );
-		$this->add_shortcode( 'sitevideoroom', array( $site_video_controller, 'sitevideo_shortcode' ) );
+		add_shortcode( self::SHORTCODE_TAG . 'sitevideoroom', array( $site_video_controller, 'sitevideo_shortcode' ) );
```
Comment (alecsammon) [inline my-video-room/module/sitevideo/class-mvrsitevideo.php:80]: as before re. constant

File: my-video-room/shortcode/class-uservideopreference.php [hunk 5]
```diff
@@ -31,7 +31,7 @@ class UserVideoPreference extends Shortcode {
 	 * Provide Runtime
 	 */
 	public function init() {
-		$this->add_shortcode( '_choose_settings', array( $this, 'choose_settings_shortcode' ) );
+		add_shortcode( self::SHORTCODE_TAG . 'choose_settings', array( $this, 'choose_settings_shortcode' ) );
```
Comment (alecsammon) [inline my-video-room/shortcode/class-uservideopreference.php:34]: as before re. constant

**Review by alecsammon — CHANGES_REQUESTED** [review]

File: my-video-room/module/sitevideo/class-mvrsitevideo.php (hunk 4 above)
Comment (ccfredbaum) [reply my-video-room/module/sitevideo/class-mvrsitevideo.php:80]: done

File: my-video-room/shortcode/class-uservideopreference.php (hunk 5 above)
Comment (ccfredbaum) [reply my-video-room/shortcode/class-uservideopreference.php:34]: done

File: my-video-room/module/security/shortcode/class-securityvideopreference.php (hunk 3 above)
Comment (ccfredbaum) [reply my-video-room/module/security/shortcode/class-securityvideopreference.php:33]: done

File: my-video-room/library/class-logger.php (hunk 2 above)
Comment (ccfredbaum) [reply my-video-room/library/class-logger.php:15]: changed

File: my-video-room/class-sitedefaults.php (hunk 1 above)
Comment (ccfredbaum) [reply my-video-room/class-sitedefaults.php:80]: done

**Review by alecsammon — APPROVED** [review]

**Pull request merged**